Drafts/README.md

```diff
@@ -1,5 +1,7 @@
-# Design-Docs
+# Drafts Archive
 
-Design Documents, Architecture etc. for SCS and related technology
+**DEPRECATION NOTICE**
```

Comment:
```suggestion
## Deprecation Notice
```
This avoids markdownlint from complaining, thinking we have used bold in place of a header format.